src/features/cli/Cli.jsx

```diff
@@ -24,28 +67,46 @@ export default function Cli({ adb }) {
       socket.stdout.pipeTo(new WritableStream({
         write: (chunk) => {
           terminal.write(chunk);
+          if (remoteShellSocket.current)
+            remoteShellSocket.current.send(chunk);
         },
       }));
-
+      
```

Comment:
Please remove unnecessary white-spaces.

Reply:
I took the liberty to add the "no-trailing-spaces" eslint rule